nexus_constructor/model/component.py

```diff
@@ -46,6 +46,17 @@ def _normalise(input_vector: QVector3D) -> Tuple[QVector3D, float]:
     return input_vector.normalized(), magnitude
 
 
+def _get_shape_group_for_pixel_data(pixel_data: Type[PixelData]) -> str:
```

Comment:
The type hint here should be `PixelData` not `Type[PixelData]`. `Type` is used when you pass a class object rather than an instance of a class.